python_modules/libraries/dagster-aws/dagster_aws/ecs/tasks.py

```diff
@@ -82,6 +81,7 @@ def default_ecs_task_definition(
                 "image": image,
                 "entryPoint": [],
                 "command": command if command else [],
+                "dependsOn": [],  # Remove any other container dependencies as well
```

Comment:
This looks good for fixing the particular error but I think we might want to reconsider the original change.

Instead of removing all containers, what if we kept the ones that are explicit dependencies? That would let us have the best of both worlds:

- intentional sidecar containers would get copied foward
- unrelated containers (like if you happen to be running dagit and daemon together but daemon doesn't depend on dagit) would get removed.